khelo/lib/ui/flow/intro/intro_screen.dart

```diff
@@ -20,38 +21,41 @@ class _IntroScreenState extends ConsumerState<IntroScreen> {
   Widget build(BuildContext context) {
     return AppPage(
       body: Builder(builder: (context) {
-        return Padding(
-          padding: context.mediaQueryPadding +
-              const EdgeInsets.symmetric(horizontal: 16.0),
-          child: SizedBox(
-            width: double.infinity,
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.center,
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: [
-                Text(
-                  context.l10n.intro_get_start_text,
-                  style: AppTextStyle.header1
-                      .copyWith(color: context.colorScheme.textPrimary),
-                  textAlign: TextAlign.center,
+        return IntroGradientBackground(
+          child: Padding(
+            padding: context.mediaQueryPadding +
+                const EdgeInsets.symmetric(horizontal: 16.0),
+            child: Center(
+              child: SingleChildScrollView(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    Text(
+                      context.l10n.intro_get_start_text,
+                      style: AppTextStyle.header1
+                          .copyWith(color: context.colorScheme.textPrimary),
+                      textAlign: TextAlign.center,
+                    ),
+                    const SizedBox(
+                      height: 16,
+                    ),
+                    Text(
+                      context.l10n.intro_description_text,
+                      style: AppTextStyle.body1
+                          .copyWith(color: context.colorScheme.textSecondary),
+                      textAlign: TextAlign.center,
+                    ),
+                    const SizedBox(
+                      height: 24,
+                    ),
+                    PrimaryButton(
+                      context.l10n.intro_sign_in_btn_text,
+                      onPressed: () => AppRoute.phoneLogin.push(context),
```

Comment:
I think we need to use pushReplacement because we will not be popping back after navigating to the login screen